content/kubernetes/active-active/preview/edit-clusters.md

```diff
@@ -0,0 +1,193 @@
+  {{<note>}}The file should contain a secret for each existing participating cluster.{{</note>}}
+
+    The example below shows a file (`all-rec-secrets.yaml`) holding secrets for two participating clusters:
+
+    ```yaml
+    apiVersion: v1
+    data:
+      password: 
+      username: 
+    kind: Secret
+    metadata:
+      name: redis-enterprise-rerc1
+    type: Opaque
+
+    ---
+
+    apiVersion: v1
+    data:
+      password: 
+      username: 
+    kind: Secret
+    metadata:
+      name: redis-enterprise-rerc2
+    type: Opaque
+
+    ```
+
+1. Get the REC credentials secret for the new participating cluster.
+
+    ```sh
+    kubectl get secret -o yaml <rec-name>
+    ```
+
+    The admin credentials secret for an REC named `rec1` has this format:
+
+    ```yaml
+    apiVersion: v1
+    data:
+      password: ABcdef12345
+      username: GHij56789
+    kind: Secret
+    metadata:
+      name: rec1
+    type: Opaque
+    ```
+
+1. Add the username and password to the new secret for that REC and namespace.
+
+    This example shows the collected secrets file (`all-rec-secrets.yaml`) for `rec1` in namespace `ns1` and `rec2` in namespace `ns2`.
+
+    ```yaml
+    apiVersion: v1
+    data:
+      password: ABcdef12345
+      username: GHij56789
+    kind: Secret
+    metadata:
+      name: redis-enterprise-rerc1
+    type: Opaque
+
+    ---
+
+    apiVersion: v1
+    data:
+      password: KLmndo123456
+      username: PQrst789010
+    kind: Secret
+    metadata:
+      name: redis-enterprise-rerc2
+    type: Opaque
+
+    ```
+
+1. Apply the file of collected secrets to every participating REC.
+
+    ```sh
+    kubectl apply -f <all-rec-secrets-file>
+    ```
+
+ If the admin credentials for any of the clusters change, update and reapply the file to all clusters.
+
+### Create RERC
+
+1. From one of the existing participating clusters, create a `RedisEnterpriseRemoteCluster` (RERC) custom resource file for the new participating cluster. 
+
+  This example shows a RERC custom resource for an REC named `rec3` in the namespace `ns3`. 
+
+  ```yaml
+  apiVersion: app.redislabs.com/v1alpha1
+  kind: RedisEnterpriseRemoteCluster
+  metadata:
+    name: rerc3
+  spec:
+    recName: rec3
+    recNamespace: ns3
+    apiFqdnUrl: test-example-api-rec3-ns3.redis.com
+    dbFqdnSuffix: -example-cluster-rec3-ns3.redis.com
+    secretName: redis-enterprise-rerc3
+  ```
+
+1. Create the RERC custom resource. 
+
+  ```sh
+  kubectl create -f <new-RERC-file>
+  ```
+
+1. Check the status of the newly created RERC custom resource.
+
+  ```sh
+  kubectl get rerc <RERC-name>
+  ```
+
+  The output should look like this: 
+  ```sh
+    NAME        STATUS   SPEC STATUS   LOCAL
+  rerc3   Active   Valid         true
+  ```
+
+### Edit REAADB spec
+
+1. Add the new RERC name to the `participatingClusters` list in the REAADB spec.
+
+  ```sh
+  kubectl patch reaadb < --type merge --patch '{"spec": {"participatingClusters": [{"name": "rerc3"}]}}'
```

Comment:
maybe add a note that this patch is adding the RERC custom resource named `rerc3`, so it will be clear that the customer should edit the kubectl command.

Reply:
also, this command is broken
it should be:
kubectl patch reaadb example-aadb-1 --type merge --patch '{"spec": {"participatingClusters": [{"name": "rerc3"}]}}'

and there should be notes that this will patch the REDB named: `example-aadb-1` with the RERC named `rerc3`